creme/creme_core/tests/management/test_spellcheck.py

```diff
@@ -0,0 +1,121 @@
+import array
+from unittest import TestCase
+
+from creme.creme_core.management.commands.i18n_spellcheck import (
+    CremePoTokenizer,
+)
+
+
+class CremePoTokenizerTestCase(TestCase):
+    def assertTextEqual(self, text, expected):
+        tokenizer = CremePoTokenizer(text)
+        self.assertFalse(tokenizer.looks_like_html(text), f"Text was: {text}")
+        tokenizer.preprocess_text()
+        self.assertEqual(expected, tokenizer._text, f"Text was: {text}")
+
+    def assertHtmlEqual(self, html, expected):
```

Comment:
There is already a 'assertHTMLEqual()' method, so this could be confusing => 'assertIsHTMLMessage()' or something like that (& rename assertTextEqual() to be consistent).